package.json

```diff
@@ -89,6 +89,7 @@
     "mocha": "^10.0.0",
     "prettier": "^2.7.1",
     "rimraf": "^3.0.2",
+    "shx": "^0.3.4",
```

Comment:
As we just need `mv`, I think we can find a simpler/smaller package for that.

I can't recall the name but I'm fairly certain there's a tiny (Windows-compatible) `mv` CLI package, sorta like `rimraf`.

Reply:
When I'm done with research, I found only this one. 
I can write a script in JavaScript if you prefer it

Reply:
[`move-cli`](https://npm.im/move-cli) perhaps? I guess alternatively we could drop `rimraf` for `shx`?

Reply:
I've replaced rimraf with `shx`